paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -119,7 +119,9 @@ public class PaparazziPlugin @Inject constructor(
     }
 
     extension.onVariants { variant ->
-      val variantSlug = variant.name.capitalize(Locale.US)
+      val variantSlug = variant.name.replaceFirstChar {
+        if (it.isLowerCase()) it.titlecase(Locale.US) else it.toString()
```

Comment:
nit: extract in a StringUtils.kt [here](https://github.com/cashapp/paparazzi/tree/ceb6cdb8f76aa056730deeec012864dad3960d46/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/utils) similar to [SqlDelight](https://github.com/sqldelight/sqldelight/blob/73070ed95b2d7bf97c987e5b54a46a36521ae596/sqldelight-compiler/src/main/kotlin/app/cash/sqldelight/core/StringUtil.kt#L3)

Reply:
Will do!